pkg/manager/manager.go

```diff
@@ -407,19 +420,19 @@ func (mngr Manager) Run() (scorecardAppUrl string, err error) {
 				mysqlCredentials,
 				dateRange,
 				minorThreshold,
-				majorThreshold)
+				majorThreshold,
+				scorecardAppUrl)
 			if err != nil {
-				return scorecardAppUrl, fmt.Errorf("error processing scorecard Run %q", err)
+				return fmt.Errorf("error processing scorecard Run %q", err)
 			}
 		}
 	}
-	return scorecardAppUrl, nil
+	return nil
 }
 
 var myScorecardManager = Manager{}
 
-func NewScorecardManager(environmentFile, documentId string) (*Manager, error) {
-	myScorecardManager.environmentFile = environmentFile
+func NewScorecardManager(documentId string) (*Manager, error) {
 	myScorecardManager.ScorecardCB = nil
 	myScorecardManager.cb = &cbConnection{}
 	myScorecardManager.documentId = documentId
```

Comment:
Would it make sense to change this (and the associated `Manager` struct) like so:

```suggestion
func newScorecardManager() (*Manager, error) {
	myScorecardManager.cb = &cbConnection{}
```

If we're moving the documentID to be something passed to `Run()` it doesn't seem to make sense to keep it in the struct. Similarly - it doesn't look like the `manager.ScorecardCB` is used elsewhere in the package so we could cull it from here to simplify the struct & constructor.

Doing these changes would mean each manager would be in charge of a reusable Couchbase connection but could be used for multiple documentID's.

Reply:
It is used a lot of places. Every fetch of a subdocument uses it as well as the upsert. I think of it as an instance variable which would put it back in the struct for the exact reason that each manager does keep a database connection open on behalf of a document. That is because we are using fetchs of subdocuments and upserts of subdocuments which are much more efficient because they go through the data service. I don't there is any really good reaso to share those connections. The manager also has a mysql connection that it keeps open on behalf of a given document and that one makes zillions of queries so it makes good sense for the manager to keep its own mysql connection open as well. Same mechanism works because both connections are kept like instance variables.

Reply:
They are both kept in a DbConnection structure, actually.

Reply:
Given the comments in other threads, I agree it makes sense to keep the `documentID`.

I'm still not sure if we need the `ScorecardCB` struct field/instance variable. Looking at the rest of the branch I'm not seeing the manager struct's `scorecardCB` being used. There's a `scorecardCB` local variable in the test suite that's used but it's not part of a manager. And I'm not seeing `manager.go` use that field. It looks like `getSubDocument()` uses the `mngr.cb` connection but puts the data back into an interface that's unrelated to manager.scorecardCB.

Reply:
`ScorecardCB` seems vestigial so we'll remove it.